packages/eslint-plugin/src/rules/member-delimiter-style.ts

```diff
@@ -83,9 +143,6 @@ export default util.createRule<Options, MessageIds>({
             },
             additionalProperties: false,
           },
-          multilineDetection: {
-            enum: ['brackets', 'last-member'],
-          },
```

Comment:
why have you removed this from the schema entirely?

Reply:
Fixed here: 75e83de73f53175e9d38a6f47ae1ff46c1b0c853